routers/root.py

```
@@ -537,8 +541,12 @@ def chat_explore_route(request: Request):
 
 
 @app.get("/chat/{integration_name}-{integration_id}/")
+@app.get("/chat/{integration_name}-{integration_id}/share/{conversation_id}/")
 def chat_route(
-    request: Request, integration_id: str = None, integration_name: str = None
+    request: Request,
+    integration_id: str | None = None,
+    integration_name: str | None = None,
```

Ruff (0.14.10)

Unused function argument: `integration_name`

(ARG001)